snmp/datadog_checks/snmp/data/conf.yaml.example

```
@@ -130,6 +136,16 @@ instances:
     #
     # retries: 5
 
+    ## @param discovery_interval - integer - optional - default: 3600
+    ## Interval between 2 discovery starts, in seconds.
+    #
+    # discovery_interval: 3600
+
+    ## @param discovery_pause - integer - optional - default: 1
+    ## Pause happening between each discovery steps, in seconds.
```

(minor) May be more explicit:

```suggestion
    ## Pause between the discovery of each network host, in seconds.
```

---

Done.